### ISSUE TITLE
Annoying hanging with synth12-64 everything

### ISSUE THREAD

**Issue by ericbsd** [jrmarino/synth#125]
I know the port is that py35-requests do not exist, but could synth skip it and avoid building or just exit with an error. I use synth with Jenkins and lately, it has been so annoying to start a build and it hangs for hours without knowing until I check the status of the build and see that it never started and it is hanging. At least, if it could exit with an error I could have an email to warn me.

```
synth12-64 everything
Scanning entire ports tree.
 progress: 45.97%              
culprit: graphics/sverchok-blender
Scan aborted because dependency could not be located.
py35-requests>0:www/py-requests@py35 (required dependency of graphics/sverchok-blender) does not exist.
 progress: 46.15%
```

**Comment by jrmarino** [comment]
exiting with an error is the only possibility.
I'm aware of the behavior, but it only happens on a corrupt ports tree (e.g. whenever the index is broken or if somebody is forking the main ports tree and doesn't ensure integrity).

In any case "skipping" is not an option.

**Comment by ericbsd** [comment]
I will agree with you on that skipping would not an option. Exiting with the error would be a nice option.

**Comment by jrmarino** [comment]
fixed with release 2.04: https://github.com/jrmarino/synth/releases/tag/v2.04

(don't forget to fix missing file on plist when upgrading on freebsd ports)

**Comment by ericbsd** [comment]
Thanks @jrmarino
